ebos/ecloutputblackoilmodule.hh

```diff
@@ -1772,13 +1924,9 @@ private:
 
     }
 
-    void createLocalFipnum_()
+    void createLocalRegion_(std::vector<int>& region)
     {
-        const auto& gridView = simulator_.vanguard().gridView();
-        if (simulator_.vanguard().eclState().fieldProps().has_int("FIPNUM"))
-            fipnum_ = simulator_.vanguard().eclState().fieldProps().get_int("FIPNUM");
-        else
-            fipnum_.resize(gridView.size(/*codim=*/0), 1);
+        //const auto& gridView = simulator_.vanguard().gridView();
```

Comment:
Either remove this disabled code or reinstate it.

Reply:
OK